java/src/server/main/Config.java

```diff
@@ -12,4 +12,5 @@
     public static IFacade facade = ServerFacade.getInstance();
     public static String host = "localhost";
     public static int port = 8081;
+    public static String persistenceLoc = "";
```

Comment:
It's for the command line parameter specifying the db to use. I didn't know if we were just going to pass in the name or location so I just defaulted to the location.